proto/supplyfinder.proto

```diff
@@ -18,7 +22,7 @@ service Supplier {
   // Sends a greeting
   rpc SayHello (HelloRequest) returns (HelloReply) {}
 
-  rpc CheckVendor (FoodID) returns (VendorInfo) {}
+  rpc CheckVendor (FoodID) returns (stream VendorInfo) {}
```

Comment:
It doesn't matter now since this is just a starter project but FYI this is a breaking change.